src/internal_events/docker_logs.rs

```diff
@@ -197,7 +197,7 @@ pub struct DockerLogsReceivedOldLogError<'a> {
 impl InternalEvent for DockerLogsReceivedOldLogError<'_> {
     fn emit(self) {
         error!(
-            message = "Received older log.",
+            message = "Received out of order log message.",
```

Comment:
Small thing but, :thought_balloon: Should we update the name of the event to more closely match this? E.g. `DockerLogsReceivedOutOfOrderError` or just `DockerLogsOutOfOrderError` ?